core/deployment/src/main/java/io/quarkus/deployment/steps/SubstrateAutoFeatureStep.java

```diff
@@ -241,12 +253,12 @@ SubstrateOutputBuildItem generateFeature(ClassOutputBuildItem output,
                             ofMethod(Class.class, "getDeclaredField", Field.class, String.class), clazz, tc.load(field));
                     tc.writeArrayValue(farray, 0, fhandle);
                     tc.invokeStaticMethod(
-                            ofMethod("org/graalvm/nativeimage/RuntimeReflection", "register", void.class, Field[].class),
+                            ofMethod(RUNTIME_REFLECTION, "register", void.class, Field[].class),
                             farray);
                 }
             }
             CatchBlockCreator cc = tc.addCatch(Throwable.class);
-            //cc.invokeVirtualMethod(ofMethod(Throwable.class, "printStackTrace", void.class), cc.getCaughtException());
+            cc.invokeVirtualMethod(ofMethod(Throwable.class, "printStackTrace", void.class), cc.getCaughtException());
```

Comment:
Is this meant to stay, or a debug thing that should stay commented out ?

Reply:
I don't know really.  It does produce an exception from Jackson.  But I think that might be a legitimate problem.